### ISSUE TITLE
shadow props doesn't work with <Image>

### ISSUE THREAD

**Issue by yozman** [facebook/react-native#25668]
React Native version:

```sh
info Fetching system and libraries information...
System:
    OS: macOS 10.14.5
    CPU: (4) x64 Intel(R) Core(TM) i5-4690 CPU @ 3.50GHz
    Memory: 2.78 GB / 20.00 GB
    Shell: 5.3 - /bin/zsh
  Binaries:
    Node: 12.4.0 - ~/.nvm/versions/node/v12.4.0/bin/node
    Yarn: 1.16.0 - ~/.nvm/versions/node/v12.4.0/bin/yarn
    npm: 6.9.0 - ~/.nvm/versions/node/v12.4.0/bin/npm
  SDKs:
    iOS SDK:
      Platforms: iOS 12.2, macOS 10.14, tvOS 12.2, watchOS 5.2
    Android SDK:
      API Levels: 27, 28, 29
      Build Tools: 27.0.3, 28.0.3, 29.0.0
      System Images: android-27 | Intel x86 Atom_64, android-28 | Intel x86 Atom_64, android-29 | Intel x86 Atom_64
  IDEs:
    Android Studio: 3.4 AI-183.6156.11.34.5522156
    Xcode: 10.2.1/10E1001 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.8.6 => 16.8.6 
    react-native: 0.60.0 => 0.60.0
```

## Steps To Reproduce

1. init with v0.60
2. add shadow props to `<Image/>`

Describe what you expected to happen:

Snack, code example, or link to a repository:

**Comment by RobinCsl** [comment]
Hi @yozman,

You would need to give more information about your situation for anyone to be able to properly help you.

If you look at the [documentation](https://facebook.github.io/react-native/docs/shadow-props#props) for shadow props on `Image` component, you see that they are only supported on iOS.
On Android, the equivalent is the `elevation` prop, which is only available on `View` component: https://facebook.github.io/react-native/docs/view-style-props#elevation

Hope that helps.

**Comment by yozman** [comment]
@RobinCsl 
it doesn't work on ios.
this is step to reproduce

1. init with v0.60
2. add shadow props to <Image/>

**Comment by RobinCsl** [comment]
@yozman Your explanations are very minimal, so please help me help you by providing more context. A code example/Expo snack is always welcome.

In this [Snack](https://snack.expo.io/@robincsl/great-apples), it looks like you're possibly missing `overflow: 'visible'` in the styles applied to your `Image`.

**Comment by yozman** [comment]
@RobinCsl 
thanks for your answer,
after add `overflow: 'visible'`
shadow is work well
but `borderRadius` is broke,
if I remove `overflow: 'visible'`
shadow not work
`borderRadius` is well

**Comment by RobinCsl** [comment]
I'm not sure if that's because React Native is unable to compute everything right (you can sometimes get some warnings in the console if you choose to *Debug JS remotely*).
In this other [Snack](https://snack.expo.io/@robincsl/great-apples-2), there is a rounded image with drop shadow thanks to a wrapping `View` around the `Image` component.

**Comment by yozman** [comment]
@RobinCsl 
yeah, I solved use [`<ImageBackground/>`](https://facebook.github.io/react-native/docs/imagebackground) instead.
but I think this is a bug with `<Image/>`

**Comment by dulmandakh** [comment]
Please provide demo code so we can investigate the issue. I'll re-open the issue once I have demo code. Thank you

**Comment by ChuksFestus** [comment]
import "intl";
import "intl/locale-data/jsonp/en";

**Comment by xi1570-krupeshanadkat** [comment]
@dulmandakh Demo Code : 
```js
import React, { Component } from 'react'
import { StyleSheet, Text, View, AppRegistry, Image } from 'react-native'

class FlexboxExamples extends Component {
  render() {
    return (
      <View style={styles.container}>
        <Image style={styles.img} source={{uri: 'https://image.freepik.com/free-vector/isometric-hiring-process_52683-44138.jpg'}} />
      </View>
    )
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: 'stretch'
  },
  img: {
    height: 240,
    resizeMode: 'cover',
    borderBottomLeftRadius: 16,
    borderBottomRightRadius: 16,
    shadowColor: '#202020',
    shadowOffset: {width: 0, height: 0},
    shadowRadius: 5,
    overflow: "visible"
  },
})

export default FlexboxExamples;
```

I just created project with CRNA and replaced content with above in App.js file. Note : Shadow properties doesn't show up on `<Image>` element unless I give `overflow: "visible"` And If i add overflow in style then Border Radius properties doesn't work. **This is bug with `<Image>` element & still present**, kindly reopen the issue :)

**Comment by necolas** [comment]
IIRC, using shadow styles on an Image instead applies the shadows as a drop-shadow filter to semi-transparent pixels in the image.
